src/main/java/com/programmers/baseball/Baseball.java

```diff
@@ -0,0 +1,60 @@
+package com.programmers.baseball;
+
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class Baseball {
+    public static void main(String[] args) {
+        final int NUMBER_SIZE = 3;
+        int gameOnGoing = 1;
+
+        while (gameOnGoing==1) {
+            Random random = new Random();
+            int[] computerNumber = IntStream.generate(() -> random.nextInt(9) + 1).distinct().limit(NUMBER_SIZE).toArray();
```

Comment:
👍 
스트림은 한줄로 모두 쓰게 되면, 가독성이 떨어지긴 합니다.
그래서 한줄에 한점이라는 케이스로 저는 많이 사용하는데요. 이와 같이 말이죠.

```suggestion
            int[] computerNumber = IntStream.generate(() -> random.nextInt(9) + 1)
            .distinct()
            .limit(NUMBER_SIZE)
            .toArray();
```

아울러 1주차 피드백때처럼, Array가 아닌 더 의미있는 Collections를 한번 사용해보면 어떨까요?